extensions/brave-search-with-results/package.json

```diff
@@ -11,11 +11,18 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "index",
+      "name": "webSearch",
```

Comment:
Can we keep the old `name`? It is the unique ID of the command where ranking, aliases, and hotkeys are saved. You can, of course, change the title 🙂